app/javascript/components/service-request-default/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import MiqFormRenderer, { useFormApi } from '@@ddf';
+import PropTypes from 'prop-types';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import createSchema from './service-request-default.schema';
+
+// NOTE: parameters to be used as filters
+let daysAgo;
+let types;
+
+const requestTypeDefault = [
+  'provision_via_foreman',
+  'template',
+  'clone_to_vm',
+  'clone_to_template',
+  'orchestration_stack_retire',
+  'provision_physical_server',
+  'physical_server_firmware_update',
+  'service_retire',
+  'service_reconfigure',
+  'clone_to_service',
+  'vm_cloud_reconfigure',
+  'vm_migrate',
+  'vm_reconfigure',
+  'vm_retire',
+];
+
+// NOTE: processing the user selected filter values
+const onSubmitTest = (values, miqRequestInitialOptions) => {
```

Comment:
Why is this method named with `Test`?  Is that a common pattern in JS?  It's just confusing cause it looks like test code as opposed to real code.

Reply:
That's not supposed to be a test function even if it looks like it 😆 renaming now